mmdet/datasets/pipelines/transforms.py

```diff
@@ -2739,3 +2739,268 @@ def __repr__(self):
         repr_str += f'saturation_delta={self.saturation_delta}, '
         repr_str += f'value_delta={self.value_delta})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class SimpleCopyPaste:
+    """Simple Copy-Paste is a Strong Data Augmentation Method for Instance
+    Segmentation The simple copy-paste transform steps are as follows:
+
+    1. The image_dest is resized keaping ratio and cropped and padded.
+    2. Randomly select an image_src, which is resized and cropped
+       and padded similarly.
+    3. Randomly select some objects from the image_src.
+    4. Paste those objects to image_dest directly, for image_dest
+       and image_src have the same size.
+    5. Update object masks of image_dest, for some origin objects
+       may be occluded.
```

Comment:
for some origin objects may be occluded. -> some origin objects may be occluded.